observations/obs_converters/SIF/work/SIF.input.txt

```diff
@@ -0,0 +1,2 @@
+../data/SIF005clip_201807.nc
+../data/SIF005clip_201808.nc
```

Comment:
Grrrr .... this is a mistake ... I am tempted to check out the 'main' version of this file, but am unclear what will happen when we go to merge 'clm_main' onto 'main'.  Any ideas?  I believe 'git remove' from my branch will not be the right solution when it comes time to merge 'clm_main' onto 'main'.